lib/TAP/Parser/YAMLish/Reader.pm

```diff
@@ -118,7 +118,8 @@ sub _read_qq {
 
     $str =~ s/\\"/"/gx;
     $str =~ s/ \\ ( [tartan\\favez] | x([0-9a-fA-F]{2}) ) 
-                 / (length($1) > 1) ? pack("H2", $2) : $UNESCAPES{$1} /gex;
+                 / (length($1) > 1) ? pack("H2", utf8::unicode_to_native($2))
```

Comment:
This is only available on recent versions right? So it should probably be a no-op on older versions?